### PR TITLE
Issue/5

### LINKED ISSUE #5: Switch from HTTP library to Express framework
Apparently it's _not_ recommended to use HTTP library directly, so we might as well use a proper framework for that. It seems simple and lightweight enough so shouldn't be too hard: https://levelup.gitconnected.com/what-is-the-difference-between-the-node-http-library-express-and-nestjs-df6a11ca0e48#:~:text=The%20feature%20provided%20by%20the,small%20and%20medium%2Dsized%20projects.

### PR DESCRIPTION
**Pull request by timberhill**
Closes #5

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,30 +1,26 @@
-const http = require("http");
+'use strict';
+const express = require('express');
+const bodyParser = require('body-parser')
 const comment = require("./comment.js");
 
-const host = 'localhost';
-const port = 8000;
+const PORT = 8080;
```
Comment (fratcliffe) [inline src/main.js:6]: Either this needs changing back to `8000` or the command in the README needs updating to `8080` 😁

**Review by fratcliffe — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 1 above)
Comment (fratcliffe) [reply src/main.js:6]: other than that it's all good I think 😄

Comment (timberhill) [reply src/main.js]: Done!

There is another PR incoming with the dockerfile though :)

**Review by fratcliffe — APPROVED** [review]
🌮

**Pull request merged**